web-wallet/src/lib/components/ExecuteMigration/ExecuteMigration.svelte

```diff
@@ -34,7 +37,7 @@
   /** @param {number} id - the chain id of the selected smart contract */
   async function handleMigration(id) {
     const txHash = await migrate(
-      amount.toString(),
+      parseUnits(amount, decimals),
```

Comment:
The `parseUnits` function doesn't seem to take into consideration string with decimal separators different than the dot and strings with thousands separators. (see https://github.com/wevm/viem/blob/main/src/utils/unit/parseUnits.ts#L21)
Maybe we should compose it with the clean function you wrote?

Reply:
I haven't considered this, I'll update the code.